src/ai-examples/azext_ai_examples/azext_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.0.65",
+    "azext.maxCliCoreVersion": "2.1.0"
```

Comment:
Can we remove this constraint if you are not sure about the max version requirement?

Reply:
Yes, good suggestion!